proto/decentraland/sdk/components/avatar_customization.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+import "decentraland/common/colors.proto";
+import "decentraland/sdk/components/common/id.proto";
+option (common.ecs_component_id) = 1087;
+
+// AvatarCustomizations sets all modifiers over the avatar's apparence. This 
+message PBAvatarCustomization {
+  decentraland.common.Color3 skin_color = 1;
+  decentraland.common.Color3 eyes_color = 2;
+  decentraland.common.Color3 hair_color = 3;
+  string body_shape_urn = 4;
+  // TODO: should name be here or in PlayerIdentityData?
```

Comment:
In my opinion, it's better to have it over there, _unless_ there's an optimization reason to include it here. That's perfectly legitimate.